plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -83,16 +84,18 @@ type genericScheduler struct {
 	cachedNodeInfoMap map[string]*schedulercache.NodeInfo
 }
 
-var resultCache = make(map[string][]*v1.Node)
+//var resultCache = make(map[string][]*v1.Node)
```

Comment:
Commenting out our changes for storing cached results to get avg score for original algorithm.